### DISCUSSION TITLE
Funding

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#27] [General]
All ideas about compensation, funding opportunities, and $$$.

**Comment by seekanddefine** [comment]
@avidrucker What do you think about tracking all the hours that you contribute your pay to funding my full time work so that when there are funds we can back pay you?

**Reply by avidrucker** [reply]
Sounds good, will do!

**Comment by avidrucker** [comment]
Students

**Reply by seekanddefine** [reply]
I have an English student! Woo!

**Comment by avidrucker** [comment]
Consulting clients

**Reply by seekanddefine** [reply]
I have a friend who asked me about this. I could pursue that option again. Let's discuss MICAH some time.

**Comment by avidrucker** [comment]
Publishing tools for thought / learning

**Reply by seekanddefine** [reply]
@avidrucker I'd like to hear some of these ideas. Brainstorming session?

**Reply by avidrucker** [reply]
> @avidrucker I'd like to hear some of these ideas. Brainstorming session?

Some of these include my collection of mental models, inspirational quotes, and short essays on learning experiences in my life.
